modules/optflow/samples/motempl.py

```diff
@@ -43,10 +47,10 @@ def draw_motion_comp(vis, (x, y, w, h), angle, color):
         gray_diff = cv2.cvtColor(frame_diff, cv2.COLOR_BGR2GRAY)
```

Comment:
To prevent error in `absdiff` in case of empty frame was retrieved by `cam.read()` block
```
if frame is None:
            break
```
is needed after `cam.read()`